native/src/main/java/io/ballerina/stdlib/grpc/nativeimpl/ModuleUtils.java

```diff
@@ -36,6 +36,10 @@ public class ModuleUtils {
     private ModuleUtils() {
     }
 
+    public static void setModule(Module module) {
```

Comment:
Creating a new function for the runtime just for the test case is not good right?

Reply:
Isnt there any other way we can create an environment?
Or do we have to go for interop or something?

Reply:
>Creating a new function for the runtime just for the test case is not good right?

Yes, we can remove this after https://github.com/ballerina-platform/ballerina-lang/pull/40212 is merged. Created https://github.com/ballerina-platform/ballerina-standard-library/issues/4421 for that.

> Isnt there any other way we can create an environment?
Or do we have to go for interop or something?

We should not create an `Environment` object other than in ballerina-lang codegen. It was wrong to make `Environment` a class with a public constructor. Hence we are making this an abstract class with https://github.com/ballerina-platform/ballerina-lang/pull/40212, and later will be made an interface (Doing like this not to have breaking changes).

The correct way is to use a Ballerina runtime in the tests. For now, I think we can merge this PR and later we can remove this after correctly fixing the tests. After merging https://github.com/ballerina-platform/ballerina-lang/pull/40212, we can also extend the `Environment` abstract class with a `TestEnvironment` and remove this. But that is also discouraged since we will use `io.ballerina.runtime.internal` imports in stdlib modules.